js/reducers/passageReducer.js

```diff
@@ -100,4 +101,14 @@ function passageReducer(state = initialState, action) {
   }
 }
 
-export default passageReducer;
+function stateSavingPassageReducer(state, action) {
+  if (!state) {
+    state = assignToEmpty(defaultState, restoreState());
```

Comment:
:smile: I was going to mention that you probably didn't want to change the method signature fo the `passageReducer` as that is used for testing, but looks like you are aware of that.

You might be able to do the `restoreState` call when you define `initialState`